components/var_dumper/introduction.rst

```diff
@@ -10,6 +10,7 @@ The VarDumper Component
     function that you can use instead of :phpfunction:`var_dump`.
 
 .. versionadded:: 2.6
+
```

Comment:
you should remove this empty line (and the one of the versionadded you added too). Versionadded directives do not allow an empty line between the start "tag" and the body